src/components/tag/tag.md

```diff
@@ -20,15 +20,15 @@ This component's accessibility was built using the [WAI ARIA Examples](https://w
 
 ### Basic examples
 
-```
+```js padded
```

Comment:
How about `padded`? 😅

Reply:
This isn't doing anything. I don't get it. It is suppose to add padding between examples but I don't know how to get it to work properly.